packages/flutter_tools/templates/app/android.tmpl/.gitignore

```
@@ -5,6 +5,7 @@ gradle-wrapper.jar
 /gradlew.bat
 /local.properties
 GeneratedPluginRegistrant.java
+.cxx
```

nit:
```suggestion
.cxx/
```

It feels a bit cleaner to be explicit that this is a directory. (And I see the versions added to the test and example apps already have that.)